src/qt/sendcoinsdialog.h

```diff
@@ -54,6 +54,9 @@ public Q_SLOTS:
     void setBalance(const CAmount& balance, const CAmount& unconfirmedBalance, const CAmount& immatureBalance,
                     const CAmount& watchOnlyBalance, const CAmount& watchUnconfBalance, const CAmount& watchImmatureBalance);
 
+Q_SIGNALS:
+    void transactionSent();
```

Comment:
If the idea is to highlight the transaction, adding the txid or like can help.

Reply:
I can extract the tx id and pass it along as a string. The part I couldn't figure out is how to convert that to whatever `focusTransaction(const QModelIndex &idx)` needs.

Reply:
Nit, rename to `coinsSent()`.

Regarding row focus, see https://stackoverflow.com/a/35154534.

Reply:
Done